lib/tasks/cucumber.rake

```diff
@@ -1,65 +1,13 @@
-# IMPORTANT: This file is generated by cucumber-rails - edit at your own peril.
-# It is recommended to regenerate this file in the future when you upgrade to a
-# newer version of cucumber-rails. Consider adding your own code to a new file
-# instead of editing this one. Cucumber will automatically load all features/**/*.rb
-# files.
```

Comment:
How does this manual editing interact with this being autogenerated again?

Reply:
I doubt we'd ever regenerate this file - the underlying interface footprint is low. Since we won't/shouldn't be adding Cucumber to any more projects, we'd have to exclude it in each project and write an explanation for why; this was just a copy/paste. We also don't seem to mind forfeiting re-auto-generation for things like `spec_helper.rb`.